internal/kldws/wsconn.go

```diff
@@ -132,9 +142,11 @@ func (c *webSocketConnection) listen() {
 		log.Debugf("WS/%s: Received: %+v", c.id, msg)
 
 		t := c.server.getTopic(msg.Topic)
-		switch msg.Type {
+		switch strings.ToLower(msg.Type) {
 		case "listen":
 			c.listenTopic(t)
+		case "listenreplies":
```

Comment:
This word feels a little awkward... not sure if there's a shorter word, can't immediately think of one though 🤷